services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/logging/CheckLoggingSpec.groovy

```diff
@@ -33,37 +30,23 @@ class CheckLoggingSpec extends BaseSpecification {
         result.hits.hits.any { hit -> hit.source.message.toLowerCase().contains(discoveryMessage) }
     }
 
-    @Issue("https://github.com/telstra/open-kilda/issues/1966")
-    @Ignore
-    def "Check Northbound, Storm and Topology Engine logging"() {
+    def "Check Northbound and Storm logging"() {
         when: "A non-existent flow is requested"
         int timeout = 180
         def flowId = "nonexistentflowid" + System.currentTimeMillis()
-        try {
-            northbound.getFlow(flowId)
-        } catch (HttpClientErrorException e) {
-
-        }
-
-        and: "Rules on a switch are validated"
-        def switchId = topology.activeSwitches.first().dpId
-        northbound.validateSwitchRules(switchId)
+        50.times { northbound.getFlow(flowId) }
```

Comment:
The `20.times` solution didn't work. I don't think that `50.times` is the way to go here. We should rethink this test/verification. Any other way to check that NB produces logs?

Reply:
Let's try it. Probably it will work.

Reply:
20 is working on virtual env fine. Also checking on hw env.

Reply:
20 and 50 are not working on staging. Reworked the test.